### PR TITLE
Add the pub/sub source

### PR DESCRIPTION
**Pull request by liferoad**
Update the pipeline to take the pub/sub topic as the source using `projects/apache-beam-testing/topics/Imagenet_openimage_50k_benchmark`.

### CODE REVIEW

File: src/pipeline.py [hunk 1]
```diff
@@ -151,7 +163,13 @@ def build_pipeline(pipeline, source_config: SourceConfig, sink_config: SinkConfi
         | "ProcessOutput" >> beam.ParDo(PostProcessor())
     )
 
-    # save the predictions to a text file
-    predictions | "WriteOutputToGCS" >> beam.io.WriteToText(  # pylint: disable=expression-not-assigned
-        sink_config.output, shard_name_template="", append_trailing_newlines=True
-    )
+    # combine all the window results into one text for GCS
+    if source_config.streaming:
+        predictions | "WriteOutputToGCS" >> beam.io.fileio.WriteToFiles(  # pylint: disable=expression-not-assigned
```
Comment (damccorm) [inline src/pipeline.py]: Can we simplify by just always using this setup?

**Review by damccorm — COMMENTED** [review]
Just had one question

File: src/pipeline.py (hunk 1 above)
Comment (liferoad) [reply src/pipeline.py]: We could but I intend to show the difference here between these two write methods. Not a lot of people are aware about this.

Comment (damccorm) [reply src/pipeline.py]: Ok, that's fine

**Review by damccorm — APPROVED** [review]

**Review by damccorm — APPROVED** [review]

**Pull request merged**